test.sh.stderr

```diff
@@ -0,0 +1,22 @@
++ truncate -s 0 dummy.img
++ truncate -s 20GiB dummy.img
++ sfdisk --force dummy.img
++ sfdisk --dump dummy.img
+++ sudo losetup --nooverlap --find --sector-size 4096 --show -- dummy.img
++ loopdev=/dev/loop0
```

Comment:
In CI it will probably be always loop0, but not necessarily when running locally. I'm not sure how to handle it nicely (one thing is to sed expected output before comparing, but it isn't very nice). So, maybe at least print a message if you've got something else than loop0 that the failure is expected?